fractions/opentracing-tracer-resolver/src/main/java/org/wildfly/swarm/opentracing/tracer/runtime/OpenTracingTracerInstaller.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2015-2016 Red Hat, Inc, and individual contributors.
```

Comment:
2018?

Reply:
Looks like the other headers need changing as well.

Reply:
There is no check for the license, other files added recently have the same header.